docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
+                    >
+                        Collapsing Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+                        If <code>true</code>, applies <code>width: auto</code> to the Table.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleCollapsing').default}
+                >
+                    <TableSampleCollapsing />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Definition Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table may be formatted to emphasize a first column
+                        that defines a row content.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleDefinition').default}
+                >
+                    <TableSampleDefinition />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Fixed Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table column&rsquo;s width can be evenly spaced.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleFixed').default}
+                >
+                    <TableSampleFixed />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Font Size
+                    </Heading>
 
-                    <TableProps props={propDefinitions} />
-                </Card>
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s default font size can be changed.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Table */}
-                <TableSample />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleFontSize').default}
+                >
+                    <TableSampleFontSize />
+                </Example>
 
```

Comment:
We lost the "Full Width" example ... was that on purpose?

Reply:
Yes.

Reply:
Okay then.  No more full width tables for _you!_  ;-) Resolving.